guacamole-ext/src/main/java/org/apache/guacamole/net/auth/User.java

```diff
@@ -93,17 +94,49 @@
      * of this user, including any active sessions. ActivityRecords
      * in this list will be sorted in descending order of end time (active
      * sessions are first), and then in descending order of start time
-     * (newer sessions are first).
+     * (newer sessions are first). If user login history is not implemented
+     * this method should throw GuacamoleUnsupportedException.
```

Comment:
Same here - I think we should specifically note that `GuacamoleUnsupportedException` may indicate that history is supported via `getHistory()`.